sal_interfaces/SALSubsystems.xml

```diff
@@ -1073,4 +1073,22 @@
     <Simulator>Internal to CSC</Simulator>
     <Configuration>https://github.com/lsst/dm_config_ocps</Configuration>
   </SALSubsystem>
+  <SALSubsystem>
+    <Name>PMD</Name>
+    <Description>Publication of data from digital devices measuring relative physical displacement as a function of time.</Description>
+    <IndexEnumeration>any</IndexEnumeration>
+    <Generics>yes</Generics>
+    <ActiveDevelopers>Eric Coughlin</ActiveDevelopers>
+    <Github>https://github.com/lsst-ts/ts_pmd</Github>
+    <JenkinsTestResults>Not Available</JenkinsTestResults>
+    <RubinObsContact>Eric Coughlin</RubinObsContact>
+    <CSCDocs/>
+    <ProductOwner>Patrick Ingraham</ProductOwner>
+    <RelatedDocuments/>
+    <SoftwareLanguage>Python</SoftwareLanguage>
+    <RuntimeLanguages>IDL</RuntimeLanguages>
+    <VendorContact>Not Applicable</VendorContact>
+    <Simulator>Not Provided</Simulator>
```

Comment:
I'm a bit surprised not to have a simulation mode. Wouldn't one be useful for exercising consumers of this data?

Reply:
Right. But I haven't implemented one yet along with actually writing any of the controller.

Reply:
I suggest making the entry correct for what you plan to support, as this makes for less "churn".

However, I also suggest figuring out if you plan to have a restricted set of CSCs. I am hoping you will decide to have one for MT and one for AT and that's it (unless there are non-MT/non-AT micrometers planned). If so, you could list those values in the appropriate tag here.

Reply:
I think having "one for AT and one for MT" is probably not the correct use case here. We may have more than one running on AT and same for MT. @patrickingraham can probably comment on this.

Reply:
My point is that one CSC can easily read multiple micrometer interface boxes. It seems more natural to me to have one CSC per telescope than one per micrometer interface box. The main thing is you need to know which micrometers is which (presumably via CSC configuration).

That assumes the micrometers will tend to have fixed location, measuring a fixed thing. If you plan to move many of them around as needed that weakens my argument.

Reply:
the micrometer boxes use a serial connection, so basically they need a computer (RPi etc) nearby.
Also, in many cases the devices will be fixed, but certainly not all, so we should keep things flexible.

Reply:
I'll change the simulator field to what I plan to support then.